### PR TITLE
Add a CLI viewer

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by wizmer** [comment]
At some point in time I'd like to migrate all NeuroM CLIs into the same 'neurom' click group.

File: apps/__main__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+def cli():
+    '''The CLI entry point'''
+
+
+@cli.command(short_help='A simple neuron viewer')
+@click.argument('input_file')
+@click.option('--plane', type=click.Choice(['3d', 'xy', 'yx', 'yz', 'zy', 'xz', 'zx']),
+              default='3d')
+@click.option('--backend', type=click.Choice(['plotly', 'matplotlib']),
+              default='plotly')
```
Comment (mgeplf) [inline apps/__main__.py]: should this really be the default when plotly is an extra?

File: apps/__main__.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+logger = logging.getLogger('morph_tool')
+logger.setLevel(logging.INFO)
+
+
+@click.group()
+def cli():
+    '''The CLI entry point'''
+
+
+@cli.command(short_help='A simple neuron viewer')
```
Comment (mgeplf) [inline apps/__main__.py]: doesn't it pull the short help from the docstring?

File: setup.py [hunk 3]
```diff
@@ -67,6 +68,9 @@
                 'apps/morph_check',
                 'apps/morph_stats',
                 ],
+    'entry_points':{
+        'console_scripts': ['neurom=apps.__main__:cli']
```
Comment (mgeplf) [inline setup.py]: This doesn't seem to work in an editable install; could be b/c of __init__.py not being there, but that will probably trigger other problems:

```
Traceback (most recent call last):
  File "/home/asdf/src/NeuroM/venv/bin/neurom", line 11, in <module>
suggestion
        'console_scripts': ['neurom=apps.__main__:cli']

    load_entry_point('neurom', 'console_scripts', 'neurom')()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named apps.__main__
```

File: apps/__main__.py (hunk 1 above)
Comment (wizmer) [reply apps/__main__.py]: Changed to matplotlib

File: apps/__main__.py (hunk 2 above)
Comment (wizmer) [reply apps/__main__.py]: Yep

File: setup.py (hunk 3 above)
Comment (wizmer) [reply setup.py]: I have added the \_\_init\_\_.py file

Comment (wizmer) [reply setup.py]: Does it work for you know (it was already working on my side)

Comment (mgeplf) [reply setup.py]: yup, seems to

**Pull request merged**